### PR TITLE
Move Knative Operator to 1.9.0

### PR DESCRIPTION
**Pull request by skonto**
- Moves operator to 1.9.0 (now on 1.6.5). Removes the patch for workload and probe overrides as it exists in 1.9.
- Required for several jiras such as: https://issues.redhat.com/browse/SRVKS-959
- The Knative upgrade process requires to move from one minor to the next but I think we are safe for the operator part (let's see).

TODO: Needs some extra typo fixes such as but not strictly required: https://github.com/knative/operator/commit/50b011c79d1859c4c8f2afcb9fe86d09d7ba19ee.

### CODE REVIEW

File: vendor/knative.dev/operator/pkg/apis/operator/base/common.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func (c *CommonSpec) GetHighAvailability() *HighAvailability {
 	return c.HighAvailability
 }
 
-// GetWorkloadOverrides implements KComponentSpec.
+// GetDeploymentOverride implements KComponentSpec.
```
Comment (skonto) [inline vendor/knative.dev/operator/pkg/apis/operator/base/common.go:209]: See todo.

File: vendor/knative.dev/operator/pkg/apis/operator/base/common.go [hunk 2]
```diff
@@ -150,7 +150,7 @@ type CommonSpec struct {
 	// +optional
 	ServiceOverride []ServiceOverride `json:"services,omitempty"`
 
-	// Override containers' resource requirements
+	// WorkloadOverride containers' resource requirements
```
Comment (skonto) [inline vendor/knative.dev/operator/pkg/apis/operator/base/common.go:153]: See todo.

**Review by ReToCode — APPROVED** [review]
/lgtm

**Pull request merged**